src/main/java/frc/robot/subsystems/hood/HoodConstants.java

```diff
@@ -1,12 +1,46 @@
 package frc.robot.subsystems.hood;
 
 import edu.wpi.first.math.geometry.Translation2d;
-import edu.wpi.first.units.Dimensionless;
-import edu.wpi.first.units.Measure;
-import edu.wpi.first.units.Units;
+import edu.wpi.first.units.*;
+import frc.robot.Constants;
+import lib.webconstants.LoggedTunableNumber;
 
 public class HoodConstants {
     public static Translation2d MECHANISM_2D_POSE = new Translation2d(1, 1);
-    public static double HOOD_LENGTH = 0.4; // m
+    public static Measure<Distance> HOOD_LENGTH = Units.Meters.of(0.4);
     public static Measure<Dimensionless> POSITION_TOLERANCE = Units.Percent.of(5);
+    public static double MAX_VELOCITY = 1; // RPS
+    public static double MAX_ACCELERATION = 4; // RPS squared
+    public static final double GEAR_RATIO = 1.0;
+    public static final double MOMENT_OF_INERTIA = 0.0003;
```

Comment:
Measure or at least add units in a comment.

Reply:
4d4a202
3b97f31

Reply:
Awesome!